component-interaction/src/app/components/user-details.component/user-details.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component, Input } from "@angular/core";
+import { UserModel } from "src/app/models/user.model";
```

Comment:
```suggestion
import { Component, Input } from "@angular/core";

import { UserModel } from "src/app/models/user.model";
```
please leave an empty line between 3rd party dependencies and your custom dependencies